README.md

```diff
@@ -196,6 +196,58 @@ describe('MSafe Core Wallet', () => {
 });
 ```
 
+### App custom parameters
+
+- You can pass custom parameters into the `appContext` parameter of the Helper.deserialize method. 
+
+```typescript
+deserialize(input: {
+    transaction: Transaction;
+    chain: IdentifierString;
+    network: SuiNetworks;
+    suiClient: SuiClient;
+    account: WalletAccount;
+    **appContext?: any;**
```

Comment:
We shall keep `appContext` in our standard. ** is a non-standard implementation. Adding appContext is supposed to deprecated the use of **